components/ai_chat/resources/page/components/conversation_list/index.tsx

```diff
@@ -114,8 +119,11 @@ function ConversationList(props: ConversationListProps) {
             id === 1 && isAIAssistant && context.shouldShowLongPageWarning
           const showEditInput = editInputId === id
           const showEditIndicator = !showEditInput && !!turn.edits?.length
-          const latestEdit = turn.edits?.at(-1);
-          const latestTurnText = latestEdit?.text ?? turn.text
+          const latestEdit =
+            (turn.edits && turn.edits.length > 0) ? turn.edits.at(-1) : null
```

Comment:
```suggestion
          const latestEdit =
            turn.edits?.at(-1)
```

Reply:
I thought it can't handle empty array, but I think I was wrong, updating! :)

Reply:
updated